src/components/application_manager/include/application_manager/application.h

```diff
@@ -443,6 +443,9 @@ class Application : public virtual InitialApplicationData,
   virtual bool is_navi() const = 0;
   virtual void set_is_navi(bool allow) = 0;
 
+  virtual bool is_rc() const = 0;
+  virtual void set_is_rc(bool allow) = 0;
```

Comment:
@BSolonenko parameter must be const

Reply:
@AByzhynar  Fixed in https://github.com/smartdevicelink/sdl_core/pull/2048/commits/3ffd281451b051e9ead29c3a877a6678109321ac